app/lib/server/functions/insertMessage.js

```diff
@@ -96,7 +96,7 @@ export const insertMessage = function(user, message, room, upsert = false) {
 	}
 
 	if (!message.ts) {
-		message.ts = new Date();
+		message.ts = Date.now();
```

Comment:
Could you explain this change? And how it will affect the clients?

Reply:
I had changed it because I noticed the date was stored as a numeric timestamp for the other imported messages, but I just realized that it's not the case for non-imported messages so I'll revert this change.